packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -4,55 +4,58 @@
     :aria-label="$gettext('Main navigation')"
     class="oc-flex oc-flex-middle"
   >
-    <oc-button
-      id="_appSwitcherButton"
-      ref="menubutton"
-      v-oc-tooltip="applicationSwitcherLabel"
-      appearance="raw-inverse"
-      variation="brand"
-      class="oc-topbar-menu-burger"
-      :aria-label="applicationSwitcherLabel"
+    <oc-expanding-dropdown
+      :expand-head="true"
+      :close-on-click="true"
+      @dropdown-open="closeSidebar"
+      @dropdown-close="reopenSidebar"
     >
-      <oc-icon name="grid" size="large" class="oc-flex" />
-    </oc-button>
-    <oc-drop
-      ref="menu"
-      drop-id="app-switcher-dropdown"
-      toggle="#_appSwitcherButton"
-      mode="click"
-      padding-size="small"
-      close-on-click
-    >
-      <oc-list class="applications-list">
-        <li v-for="(n, nid) in applicationsList" :key="`apps-menu-${nid}`" @click="clickApp(n)">
-          <oc-button
-            :key="n.url ? 'apps-menu-external-link' : 'apps-menu-internal-link'"
-            :type="n.url ? 'a' : 'router-link'"
-            :target="n.target"
-            :href="n.url"
-            :to="n.path"
-            :appearance="n.active ? 'raw-inverse' : 'raw'"
-            :variation="n.active ? 'primary' : 'passive'"
-            :class="{ 'oc-background-primary-gradient router-link-active': n.active }"
-          >
-            <span class="icon-box">
-              <oc-icon :name="n.icon" variation="inherit" />
-            </span>
-            <span v-text="$gettext(n.title)" />
-            <oc-icon v-if="n.active" name="check" class="active-check" variation="inherit" />
-          </oc-button>
-        </li>
-      </oc-list>
-    </oc-drop>
+      <template #toggle>
+        <oc-icon name="grid" size="large" class="oc-flex" />
+      </template>
+      <template #head="{ dropdownVisible }">
+        <div class="dropdown-title-wrapper">
+          <transition name="slide-fade">
+            <h3 v-if="dropdownVisible">Applications</h3>
```

Comment:
two things:
- string is not translated
- `h3` is probably not correct here. If I see it correctly then there is no `h2` in the hierarchy (`h1` should be there), so you'd need to use a `h2` tag.